nimby-vs-yimby/README.md

```diff
@@ -0,0 +1,303 @@
+del result_df['county']
+del result_df['code']
+result_df.to_csv("~/result.csv")
+
+counts_by_county = result_df['county_state'].value_counts()
+
+df_counts_by_county = pd.DataFrame.from_records([counts_by_county.to_dict()]).transpose()
+df_counts_by_county.reset_index(inplace=True)
+df_counts_by_county = df_counts_by_county.rename(columns={'index': 'county_state', 0: 'n'})
+df_counts_by_county = df_counts_by_county.sort_values('n', ascending=False)
+
+df_counts_by_county = pd.merge(df_counts_by_county, df_counties, on='county_state')
+
+df_counts_by_county['per_capita'] = df_counts_by_county['n'] / df_counts_by_county['pop2010']
+df_counts_by_county['per_area'] = df_counts_by_county['n'] / df_counts_by_county['area_2010']
+
+per_capita_mean = float(df_counts_by_county['per_capita'].mean())
+per_capita_stdev = float(df_counts_by_county['per_capita'].std())
+
+df_counts_by_county['per_capita_stdevs_from_mean'] = (df_counts_by_county['per_capita'] - per_capita_mean) / per_capita_stdev
+
+per_area_mean = float(df_counts_by_county['per_area'].mean())
+per_area_stdev = float(df_counts_by_county['per_area'].std())
+
+df_counts_by_county['per_area_stdevs_from_mean'] = (df_counts_by_county['per_area'] - per_area_mean) / per_area_stdev
+df_counts_by_county = df_counts_by_county.sort_values('per_capita', ascending=False)
+
+df_counts_by_county.to_csv("~/counts.csv")
+```
+
+The above code computed a Pandas data frame with the following fields for each country:
+* `n` - absolute number of new real estate projects (number of initial sale records).
+* `per_capita` - `n` divided by population of the county (for year 2010).
+* `per_area` - `n` divided by land area of the county (in square miles).
+* `per_capita_stdevs_from_mean` - how many standard deviations is `per_capita` away from the mean value of entire data we're looking into
+* `per_area_stdevs_from_mean` - equivalent of `per_capita` for `per_area` value.]
+
+Based on the numbers that we have now, we can find the best and worst counties in terms of real estate development.
+
+How does the overall distribution look like for per capita values and their distance from the mean? We can plot some simple bar charts to find out.
+
+
+```python
+import plotly.express as px
+from IPython.display import Image
+
+labels = {
+    "county_state": "County",
+    "per_area": "new projects per square mile",
+    "per_capita": "new projects per capita",
+    "per_capita_stdevs_from_mean": "standard devs. from mean (per capita)",
+    "per_area_stdevs_from_mean": "standard devs. from mean (per area)"
+}
+
+df_counts_by_county = df_counts_by_county.sort_values('per_capita', ascending=False)
+fig = px.bar(df_counts_by_county, x='county_state', y='per_capita', title="New real estate sales per capita", labels=labels)
+#fig.show()
+fig.write_image("per_capita_by_county.png")
+Image(filename="per_capita_by_county.png")
+```
+
+
+
+
+    
+![png](output_6_0.png)
```

Comment:
These aren't rendering for me. Also give them a good caption.

Reply:
They are rendering on Github. See: https://github.com/rl1987/data-analysis/tree/ticket11/nimby-vs-yimby

How should image files be referenced from Markdown document so that DoltHub blog system can handle them?